pkg/controller/acmeorders/sync_test.go

```diff
@@ -651,16 +651,6 @@ rUCGwbCUDI0mxadJ3Bz4WxR6fyNpBK2yAinWEsikxqEt
 
 				},
 				FakeFetchCert: func(_ context.Context, url string, bundle bool) ([][]byte, error) {
-					if url != "http://alturl" {
```

Comment:
I think this testcase might be a problem, here the passed url will be `""`, so calling FetchCert will not result in the desired result.
We probably do not want to fetch the default chain and use certChain instead.